app/api/dao/organization.py

```diff
@@ -0,0 +1,104 @@
+
+from http import HTTPStatus
+from flask import json
+from app.database.models.bit_schema.organization import OrganizationModel
+from app.database.models.bit_schema.user_extension import UserExtensionModel
+from app import messages
+from app.api.request_api_utils import AUTH_COOKIE
+from app.utils.bitschema_utils import Timezone, OrganizationStatus
+
+
+class OrganizationDAO:
+    
+    """Data Access Object for Users_Extension functionalities"""
+
+    @staticmethod
+    def get_organization(representative_id):
+        """Retrieves the organization that is represented by the user which ID is passed as parameter.
+
+        Arguments:
+            rep_id: The ID of the user who represents the organization.
+
+        Returns:
+            The OrganizationModel class represented by the user whose ID was searched.
+        """
+
+        representative_additional_info = UserExtensionModel.find_by_user_id(representative_id)
+        try:
+            if representative_additional_info.is_organization_rep:
+                result = OrganizationModel.find_by_representative(representative_id)
```

Comment:
Can this be EAFP/try-catch based? Makes more sense to me intuitively

Reply:
ok, will try and change it now.

Reply:
@ramitsawhney27 , I'm not sure how to use EAFP to check if the value of is_organization_rep is true, which is my intention here. Can you please advise?

Reply:
I meant for line 30-44. We can do:

try: 
    return {}
except ...:
    return messages. OR...

Not a compulsory thing, if you're unclear or disagree, we can skip this for now

Reply:
I see. Done. I've changed it to EAFP now 😁